ranger/ext/img_display.py

```diff
@@ -43,6 +43,12 @@
EB", "\U000006EC", "\U00000730", "\U00000732", "\U00000733", "\U00000735", "\U00000736", "\U0000073A", "\U0000073D", "\U0000073F", "\U00000740", "\U00000741", "\U00000743", "\U00000745", "\U00000747", "\U00000749", "\U0000074A", "\U000007EB", "\U000007EC", "\U000007ED", "\U000007EE", "\U000007EF", "\U000007F0", "\U000007F1", "\U000007F3", "\U00000816", "\U00000817", "\U00000818", "\U00000819", "\U0000081B", "\U0000081C", "\U0000081D", "\U0000081E", "\U0000081F", "\U00000820", "\U00000821", "\U00000822", "\U00000823", "\U00000825", "\U00000826", "\U00000827", "\U00000829", "\U0000082A", "\U0000082B", "\U0000082C", "\U0000082D", "\U00000951", "\U00000953", "\U00000954", "\U00000F82", "\U00000F83", "\U00000F86", "\U00000F87", "\U0000135D", "\U0000135E", "\U0000135F", "\U000017DD", "\U0000193A", "\U00001A17", "\U00001A75", "\U00001A76", "\U00001A77", "\U00001A78", "\U00001A79", "\U00001A7A", "\U00001A7B", "\U00001A7C", "\U00001B6B", "\U00001B6D", "\U00001B6E", "\U00001B6F", "\U00001B70", "\U00001B71", "\U00001B72", "\U00001B73", "\U00001CD0", "\U00001CD1", "\U00001CD2", "\U00001CDA", "\U00001CDB", "\U00001CE0", "\U00001DC0", "\U00001DC1", "\U00001DC3", "\U00001DC4", "\U00001DC5", "\U00001DC6", "\U00001DC7", "\U00001DC8", "\U00001DC9", "\U00001DCB", "\U00001DCC", "\U00001DD1", "\U00001DD2", "\U00001DD3", "\U00001DD4", "\U00001DD5", "\U00001DD6", "\U00001DD7", "\U00001DD8", "\U00001DD9", "\U00001DDA", "\U00001DDB", "\U00001DDC", "\U00001DDD", "\U00001DDE", "\U00001DDF", "\U00001DE0", "\U00001DE1", "\U00001DE2", "\U00001DE3", "\U00001DE4", "\U00001DE5", "\U00001DE6", "\U00001DFE", "\U000020D0", "\U000020D1", "\U000020D4", "\U000020D5", "\U000020D6", "\U000020D7", "\U000020DB", "\U000020DC", "\U000020E1", "\U000020E7", "\U000020E9", "\U000020F0", "\U00002CEF", "\U00002CF0", "\U00002CF1", "\U00002DE0", "\U00002DE1", "\U00002DE2", "\U00002DE3", "\U00002DE4", "\U00002DE5", "\U00002DE6", "\U00002DE7", "\U00002DE8", "\U00002DE9", "\U00002DEA", "\U00002DEB", "\U00002DEC", "\U00002DED", "\U00002DEE", "\U00002DEF", "\U00002DF0", "\U00002DF1", "\U00002DF2", "\U00002DF3", "\U00002DF4", "\U00002DF5", "\U00002DF6", "\U00002DF7", "\U00002DF8", "\U00002DF9", "\U00002DFA", "\U00002DFB", "\U00002DFC", "\U00002DFD", "\U00002DFE", "\U00002DFF", "\U0000A66F", "\U0000A67C", "\U0000A67D", "\U0000A6F0", "\U0000A6F1", "\U0000A8E0", "\U0000A8E1", "\U0000A8E2", "\U0000A8E3", "\U0000A8E4", "\U0000A8E5", "\U0000A8E6", "\U0000A8E7", "\U0000A8E8", "\U0000A8E9", "\U0000A8EA", "\U0000A8EB", "\U0000A8EC", "\U0000A8ED", "\U0000A8EE", "\U0000A8EF", "\U0000A8F0", "\U0000A8F1", "\U0000AAB0", "\U0000AAB2", "\U0000AAB3", "\U0000AAB7", "\U0000AAB8", "\U0000AABE", "\U0000AABF", "\U0000AAC1", "\U0000FE20", "\U0000FE21", "\U0000FE22", "\U0000FE23", "\U0000FE24", "\U0000FE25", "\U0000FE26", "\U00010A0F", "\U00010A38", "\U0001D185", "\U0001D186", "\U0001D187", "\U0001D188", "\U0001D189", "\U0001D1AA", "\U0001D1AB", "\U0001D1AC", "\U0001D1AD", "\U0001D242", "\U0001D243", "\U0001D244"]  # noqa: E501
```

Comment:
Do we really need this? I'm worried it'll be hard to keep up to date going forward.

Reply:
I agree that awful and i hate it but there is no workaround ( hopefully it will probably not change in the future). Best i can do is put in comment the bash command to generate the list.

Reply:
Maybe we can simply run the command that provides these when the method is initialized?

Reply:
I feel like it's a bit silly to parse a [file](https://unicode.org/Public/UNIDATA/UnicodeData.txt) of 1,82Mo each time the method is initialized, simply to get a list of only 256 elements. Moreover kitty will probably never change the character chosen due to the risk of breaking a lot of implementations.

Reply:
OK, fine. Can you add some more explanation as to what the diacritics are for to the comment?